app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsController.scala

```diff
@@ -193,8 +194,16 @@ class ReliefDetailsController @Inject() (
                 draftReturn.copy(
                   reliefDetailsAnswers = Some(
                     answers.fold(
-                      _.copy(privateResidentsRelief = Some(AmountInPence.fromPounds(p))),
-                      _.copy(privateResidentsRelief = AmountInPence.fromPounds(p))
+                      _.copy(privateResidentsRelief = Some(AmountInPence.fromPounds(p)), lettingsRelief = None),
+                      c =>
+                        if (c.privateResidentsRelief === AmountInPence.fromPounds(p))
```

Comment:
this check for equality should come outside before you do `draftReturn.copy` so we don't unecessarily unset the value of `lettingRelief` when the answer hasn't changed

Reply:
Done.